src/resolvers.ts

```diff
@@ -6,13 +6,23 @@ export default {
     movie: movieById
   },
   Movie: {
-    posterPath: (
-      parent: any,
-      args: any,
-      context: any
-    ): String => {
-      return `https://www.themoviedb.org/t/p/w500${parent.poster_path}`
```

Comment:
Might make this a 'const' so that it isn't repeated so many times:

const POSTER_URL_PREFIX = 'https://www.themoviedb.org/t/p/';

then:

return `${POSTER_URL_PREFIX}w500${parent.poster_path}`;

Reply:
Thanks Jaime- I agree that is better. Just made that change.